public/app/percona/integrated-alerting/components/Alerts/Alerts.tsx

```diff
@@ -124,24 +124,56 @@ export const Alerts: FC = () => {
 
   const renderSelectedSubRow = React.useCallback((row: Row<Alert>) => <pre>{row.original.rule?.expr}</pre>, []);
 
+  const handleSilenceAll = useCallback(async () => {
+    await AlertsService.toggle({ silenced: 'TRUE', alert_ids: [] });
+    getAlerts();
+  }, []);
+
+  const handleUnsilenceAll = useCallback(async () => {
+    await AlertsService.toggle({ silenced: 'FALSE', alert_ids: [] });
+    getAlerts();
+  }, []);
+
   useEffect(() => {
     getAlerts();
   }, [pageSize, pageIndex]);
 
   return (
-    <Table
-      showPagination
-      totalItems={totalItems}
-      totalPages={totalPages}
-      pageSize={pageSize}
-      pageIndex={pageIndex}
-      onPaginationChanged={handlePaginationChanged}
-      data={data}
-      columns={columns}
-      pendingRequest={pendingRequest}
-      emptyMessage={noData}
-      getCellProps={getCellProps}
-      renderExpandedRow={renderSelectedSubRow}
-    />
+    <>
+      <div className={style.actionsWrapper}>
+        <Button
+          size="md"
+          icon="bell-slash"
+          variant="link"
+          onClick={handleSilenceAll}
+          data-testid="alert-rule-template-add-modal-button"
```

Comment:
Can we have a title prop to the button with a text like "Silence this alert" to be easier to understand if you are silencing/unsilencing?

Reply:
Ok maybe I missed it somehow